harbor-core.yaml

```diff
@@ -0,0 +1,59 @@
+package:
+  name: harbor-core
+  version: 2.10.1
+  epoch: 0
+  description: An open source trusted cloud native registry project that stores, signs, and scans content
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - swagger
+  environment:
+    CGO_ENABLED: "0"
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/goharbor/harbor
+      tag: v${{package.version}}
+      expected-commit: b7b884760ae72ea246b6fdc23f2a8fd5c68610ef
+
+  - runs: |
+      mkdir -p ${{targets.destdir}}/harbor
+
+      # Copy views, migrations, and icons
+      cp -rf ./src/core/views ${{targets.destdir}}/harbor/views
+      cp -rf ./make/migrations ${{targets.destdir}}/harbor/migrations
+      cp -rf ./icons ${{targets.destdir}}/harbor/icons
+
+      # Generate API
+      swagger generate server --template-dir=tools/swagger/templates --exclude-main --additional-initialism=CVE --additional-initialism=GC --additional-initialism=OIDC -f api/v2.0/swagger.yaml -A harbor --target src/server/v2.0
+
+  - uses: go/bump
+    with:
+      modroot: ./src
+      deps: github.com/cloudevents/sdk-go/v2@v2.15.2 github.com/go-jose/go-jose/v3@v3.0.3 github.com/jackc/pgproto3/v2@v2.3.3 github.com/jackc/pgx/v4@v4.18.2 google.golang.org/protobuf@v1.33.0
```

Comment:
```suggestion
      deps: github.com/cloudevents/sdk-go/v2@v2.15.2 github.com/go-jose/go-jose/v3@v3.0.3 github.com/jackc/pgproto3/v2@v2.3.3 github.com/jackc/pgx/v4@v4.18.2 google.golang.org/protobuf@v1.33.0 helm.sh/helm/v3@v3.14.2
```

Bah, I missed this one!

Reply:
This one in particular hasn't been fixed. For packages that pull the helm go module in, an advisory was created. It was determined non-problematic for us since we aren't leveraging the helm modules debug mode

Reply:
Ah, right that CVE. How about bumping to 3.14.3 to avoid having to create an advisory for this?

Reply:
With `helm.sh/helm/v3@v3.14.3` I get a clean scan: 

```
[17:19:39|24-03-19]user@jamon: /home/user/wolfi-os
 1 % wolfictl scan packages/x86_64/harbor-core-2.10.1-r0.apk
🔎 Scanning "packages/x86_64/harbor-core-2.10.1-r0.apk"
```

Reply:
Nice! That patch must of just landed. Will update my PR real fast